x-pack/plugin/rank-rrf/src/yamlRestTest/resources/rest-api-spec/test/rrf/1000_rrf_retriever_highlighting.yml

```diff
@@ -0,0 +1,110 @@
+setup:
```

Comment:
Added a different file per test scenario for better readability, but can rework them and merge them to a single file, to decrease setting up a new index every time.